scripts/ynh_composer__3

```diff
@@ -1,8 +1,14 @@
 #!/bin/bash
 
+readonly YNH_DEFAULT_COMPOSER_VERSION=1.10.17
```

Comment:
Kimai 1.11 is compatible with Composer 2. I would have expected the opposite was: default Composer 2 with the option to switch back to Composer 1. Or is that something on the YH roadmap for a later stage? 
Obviously I am trying to keep Kimai compatible with both versions, just curious why you choose to opt in for Composer 2 instead of opt out.

Reply:
Well, I choose a default to composer 1 because most of application haven't switched to composer 2 and we have several apps that no more install.

If Kimai2 is ready for composer 2 we can switch it easily to composer 2 just initializing `YNH_COMPOSER_VERSION` in _common.sh

Reply:
just done at https://github.com/YunoHost-Apps/kimai2_ynh/pull/31/commits/a80da800d3e06de87132469cbc35467c6d98f7f7

Reply:
Nice 👍 My GitHub Action test stack uses Composer 2 while I still use Composer 1 locally. 

I assume the YH test suite install Kimai now with Composer 2 and we can see the results, right?

Reply:
Oh and btw: thanks for caring about Kimai here! 😄

Reply:
> 
> 
> Nice 👍 My GitHub Action test stack uses Composer 2 while I still use Composer 1 locally.
> 
> I assume the YH test suite install Kimai now with Composer 2 and we can see the results, right?

Yes, next test will run with composer 2.0.4, we will see what's happens

Reply:
> 
> 
> Oh and btw: thanks for caring about Kimai here! 😄

Your welcome ^^ It's because several apps started to fail because of composer 2 :/ https://ci-apps.yunohost.org/ci/apps/kimai2/